framework/src/multiapps/TransientMultiApp.C

```diff
@@ -260,7 +260,8 @@ TransientMultiApp::solveStep(Real dt, Real target_time, bool auto_advance)
 
           if (!converged)
           {
-            mooseWarning("While sub_cycling " << name() << _first_local_app+i << " failed to converge!" << std::endl);
+            mooseWarning("While sub_cycling ", name(), _first_local_app+i, " failed to converge!\n");
```

Comment:
Script just preserves things that were there that aren't related to the changes required to support this PR.